cmp/form/FormModel.js

```diff
@@ -264,7 +264,9 @@ export class FormModel extends HoistModel {
             get(target, name, receiver) {
 
                 const field = me.fields[name];
-                if (field) return field.getDataOrProxy();
+                if (field?.isFieldModel) {
+                    return field.getDataOrProxy();
+                }
```

Comment:
This was triggered by Inspector code looking to print the type of this `values` property via `values.constructor.name`. That threw in this code, which is assuming that all non-null properties accessed off of `values` will be a `FieldModel` - which is almost always true except for edge-cases like this one.